src/main/java/frc/robot/swerve/ModuleIOReal.java

```diff
@@ -131,21 +118,30 @@ public void setVelocity(double velocity) {
         )).withEnableFOC(true));
     }
 
+    @Override
+    public SwerveModuleState getModuleState() {
```

Comment:
Should use the new `AutoLogOutput` annotation.

Reply:
913f7b3969a7ab3cf238956c1e63f17228e43c48
If I understood correctly it needs to be on most of the `.get...` methods right?

Reply:
The most accurate answer is "use `@AutoLogOutput` for every function that returns a value we log as an output". But because there are many getters that return outputs then "most" is a good answer.
However, note that I was confused and should have commented that on the subsystem class, not here – after all, that's where outputs are recorded). Sorry for that. So the annotation should be moved there.

Reply:
Very sorry for the confusion again. Because swerve modules are parts of the swerve subsystem, it does make sense to log outputs from them. So `@AutoLogOutput` should be used both in `ModuleIO` _and_ in `SwerveDrive`, each for its respective outputs. Just make sure that the modules output keys are under the swerve ones (something like `.../swerve/module1/angleSetpoint`). Sorry again.